Docker/README.md

```diff
@@ -1,19 +1,25 @@
-## Notes:
-1. You can use the polar.usc.edu elastic data, or your local elastic data to begin with the process. Replace the polar URL with your local elastic index URL to do so.
-2. Add your files (pdfs) to the `data/polar/files` folder to extract insights.
+This is the Dockerized version of Polar-deep-insights project. Two parts of the project, insight-generator which is a python library used to extract information and insight-visualizer which is a javascript application used for data visualization can be installed and run on Docker containers using PDI Docker.  
 
 ## Pre-Reqs
-
+1. Install Node and NPM from [here](https://nodejs.org/en/) if you don't have it already installed. 
 1. Install elasticsearch-tools (https://www.npmjs.com/package/elasticsearch-tools)
```

Comment:
Add "You may need to sudo the command below, depending on your account permissions.